src/libraries/System.Reflection.MetadataLoadContext/src/README.md

```diff
@@ -0,0 +1,40 @@
+## About
+
+Provides read-only reflection on assemblies in an isolated context with support for assemblies that target different processor architectures and runtimes. Using MetadataLoadContext enables you to inspect assemblies without loading them into the main execution context. Assemblies in MetadataLoadContext are treated only as metadata, that is, you can read information about their members, but cannot execute any code contained in them.
+
+For more information, see the documentation:
+
+- [How to: Inspect assembly contents using MetadataLoadContext](https://docs.microsoft.com/dotnet/standard/assembly/inspect-contents-using-metadataloadcontext)
+- [System.Reflection.MetadataLoadContext](https://docs.microsoft.com/dotnet/api/system.reflection.metadataloadcontext)
+- [System.Reflection.MetadataAssemblyResolver](https://docs.microsoft.com/dotnet/api/system.reflection.metadataassemblyresolver)
+
+## Example
+
+The following example shows how print the list of types defined in an assembly.
+
+```
```

Comment:
NIT: Not sure if adding `cs` would work for md file and make the code look better
```suggestion
```cs
```

Reply:
Syntax highlighting is currently not supported in NuGet (https://github.com/NuGet/NuGetGallery/issues/8632). But it wouldn't hurt to add language indicator so syntax highlighting will light up on Github or in some third-party viewers that support it.